modules/gpgsqlbackend/gpgsqlbackend.cc

```diff
@@ -97,6 +97,7 @@ class gPgSQLFactory : public BackendFactory
     declare(suffix,"delete-names-query","","delete from records where domain_id=$1 and name=$2");
 
     declare(suffix,"add-domain-key-query","", "insert into cryptokeys (domain_id, flags, active, content) select id, $1, $2, $3 from domains where name=$4");
+    declare(suffix,"get-last-inserted-key-id-query","", "select lastval()");
```

Comment:
`lastval()` doesn't take any arguments. Do you mean replacing it with `currval(sequence)`?